### PR TITLE
docs: ALTER TABLE SET PARAM, out-of-order changes

### PR DESCRIPTION
**Pull request by bsmth**
* Minor changes to hysteresis guide
* Added `ALTER TABLE SET PARAM` docs
* Minor change to sidebar navigation, removed `Deploy QuestDB` from top-level nav as content is moved into separate docs

### CODE REVIEW

File: docs/guides/hysteresis.md [hunk 1]
```diff
@@ -36,33 +36,31 @@ the previously received value, the boundary for this data is `10 seconds` and we
...
+the boundary; the remaining rows stay in memory to be committed later.
 
-## O3 commit hysteresis parameters
+## Out-of-order (O3) commit parameters
 
-Commit hysteresis parameters are available for specifying that rows may only be
-committed when they are outside a window of time for which they are expected to
-be out-of-order. The following server configuration parameters are
-user-configurable which help with applying hysteresis practically:
+Commit parameters allow for specifying that commits of out-of-order data should
```
Comment (ideoma) [inline docs/guides/hysteresis.md:45]: Should we mention that this is only case for ILP somewhere? PgWire is committed manually and REST is committed row by row or on full csv

File: docs/guides/hysteresis.md [hunk 2]
```diff
@@ -71,35 +69,35 @@ An O3 commit will occur:
...
 
-The defaults for the hysteresis algorithm are optimized for real-world usage and
-should cover most O3 patterns for timestamp values. The default configuration is
-as follows:
+The defaults for the out-of-order algorithm are optimized for real-world usage
+and should cover most patterns for timestamp arrival. The default configuration
+is as follows:
 
 ```txt title="Defaults"
 cairo.o3.commit.hysteresis.in.ms=1000
```
Comment (ideoma) [inline docs/guides/hysteresis.md:79]: I don't think this is true. Default looks to be 300,000 ms

File: docs/guides/hysteresis.md [hunk 3]
```diff
@@ -71,35 +69,35 @@ An O3 commit will occur:
...
 cairo.o3.max.uncommitted.rows=1000
```
Comment (ideoma) [inline docs/guides/hysteresis.md:80]: Default looks to be 500,000 ms

File: docs/guides/hysteresis.md [hunk 4]
```diff
@@ -71,35 +69,35 @@ An O3 commit will occur:
...
 line.tcp.maintenance.job.hysteresis.in.ms=1000
```
Comment (ideoma) [inline docs/guides/hysteresis.md:81]: current default is 30,000

File: docs/guides/hysteresis.md (hunk 1 above)
Comment (bsmth) [reply docs/guides/hysteresis.md:45]: This is important to highlight, yes

Comment (bsmth) [reply docs/guides/hysteresis.md:46]: added in 2449588a37adf1322fac5c2d0e1107590cbb43a2

File: docs/guides/hysteresis.md (hunk 2 above)
Comment (bsmth) [reply docs/guides/hysteresis.md]: Updated

File: docs/guides/hysteresis.md (hunk 3 above)
Comment (bsmth) [reply docs/guides/hysteresis.md]: Updated in fc150aaf46df034778ec4d58fba609646984b6a8

File: docs/guides/hysteresis.md (hunk 4 above)
Comment (bsmth) [reply docs/guides/hysteresis.md]: Updated in fc150aaf46df034778ec4d58fba609646984b6a8

**Review by ideoma — APPROVED** [review]

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**